src/wm/com.solus-project.budgie.wm.gschema.xml

```diff
@@ -86,10 +86,10 @@
 			<description>This key overrides the key in org.gnome.desktop.wm.preferences when running Budgie.</description>
 		</key>
 
-		<key type="b" name="force-unredirect">
-			<default>false</default>
-			<summary>Forcibly disable screen unredirection</summary>
-			<description>Disables the screen unredirection which may improve performance with certain drivers.</description>
+		<key type="b" name="enable-unredirect">
+			<default>true</default>
+			<summary>Enable unredirection of fullscreen applications</summary>
+			<description>Enable unredirection which will allow frames to bypass compositing for fullscreen application.</description>
```

Comment:
*for fullscreen applications

Reply:
Fixed in 79d9f2ab34e17015d55e41fe3afddcf84cfc4335